src/backend/controllers/tankController.js

```diff
@@ -1,68 +1,64 @@
 // @flow strict
 
-// Flow type import
-import type { Request, Response } from 'express';
-
 const User = require('../../models/userModel');
 const Tank = require('../../models/tankModel');
 
 const { validationResult } = require('express-validator');
 
-exports.getFavorite = async (req: Request, res: Response) => {
+// Flow type import
+import type {
+	$Request,
+	$Response,
+	NextFunction,
+	Middleware,
+} from 'express';
+
+exports.getFavorite = async (req: $Request, res: $Response) => {
 	await User.findById(req.user.id, 'favoriteTankId', (err: Error, myUser: User) => {
 		if(err) {
 			res.send(err);
 			console.log('could not find user');
 		} 
-		else {
+		else
 			res.send(myUser.favoriteTankId);
-			console.log('Retrieved Favorite Tank.');
-		}
 	});
 }
 
-exports.favoriteTank = async (req: Request, res: Response) => {
+exports.favoriteTank = async (req: $Request, res: $Response) => {
 	// the 'new' option means return the document after it has been updated
-	await User.findOneAndUpdate( { _id: req.user.id }, {favoriteTankId : req.body.favoriteTankId}, {'new':true }, (err: Error, result: User) => {
+	// the 'useFindAndModify' is just so it doesnt throw a deprecation warning
+	await User.findOneAndUpdate( { _id: req.user.id }, {favoriteTank : req.body.favoriteTankId}, {'new':true, 'useFindAndModify':false}, (err: Error, result: User) => {
```

Comment:
I removed that `useFindAndModify` for a reason. It's redundant since it's used as a general option in server.js
![image](https://user-images.githubusercontent.com/24559442/77240271-6c1ab480-6bba-11ea-9997-19915c7f9f5b.png)

Unless this call still complains about this option.